index.bs

```diff
@@ -3095,7 +3096,7 @@ platform to enumerate all the authenticator's credentials so that the client can
 
 This is the first-time flow, in which a new credential is created and registered with the server.
 
-1. The user visits example.com, which serves up a script. At this point, the user must already be logged in using a legacy
+1. The user visits example.com, which serves up a script. At this point, the user MUST already be logged in using a legacy
```

Comment:
This is descriptive - not normative language. Please back this individual change out.

Reply:
done